app/parse.py

```diff
@@ -1,4 +1,13 @@
-from dataclasses import dataclass
+import csv
+import time
+import requests
+
+from dataclasses import dataclass, fields, astuple
+from bs4 import BeautifulSoup
+from urllib.parse import urljoin
+
+BASE_URL = "https://quotes.toscrape.com"
+AUTHORS_URL_SET = set()
```

Comment:
Redundant suffix _SET

Reply:
> Why you always add result_ to all names? Please explain this) I do not see why we need this

because it is a constant and even in the video Danil used upper case, if it's a bug, I'll fix it
I don't understand what this has to do with helping to implement a parallel launch.

Reply:
so it is easier to separate the global variable from the non-global variable, in the attributes of the class I use lowercase letters

Reply:
> because it is a constant and even in the video Danil used upper case,
As you say. Please read a little info documentation and check your code. 
https://intellipaat.com/blog/tutorial/python-tutorial/python-variables/#no3
https://en.wikipedia.org/wiki/Constant_(computer_programming)